app/Services/DependencyService.php

```diff
@@ -10,7 +10,7 @@
 
 class DependencyService
 {
-    public static function touchDependencyByKey(string $key, ?User $user): void
+    public static function touchDependencyByKey(string $key, ?User $user = null): void
```

Comment:
I didn’t know you needed the default - everyday is a school day!

Reply:
You don't - I just didn't want to have to specify NULL in all my patches :D